<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>538</width>
    <height>491</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Uslugi</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(231, 255, 212);
</string>
  </property>
  <widget class="QLabel" name="label_uslugi">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>20</y>
     <width>241</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cascadia Mono Light</family>
     <pointsize>27</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Услуги</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>341</y>
     <width>551</width>
     <height>161</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(170, 255, 127);</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>481</width>
     <height>178</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="2" column="0">
     <widget class="QLabel" name="label_uslugi_2">
      <property name="font">
       <font>
        <family>Cascadia Mono Light</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Услуга</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="lineEdit_usluga">
      <property name="font">
       <font>
        <family>Cascadia Mono Light</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(170, 255, 127);</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_uslugi_3">
      <property name="font">
       <font>
        <family>Cascadia Mono Light</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>ID услуги</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_uslugi_4">
      <property name="font">
       <font>
        <family>Cascadia Mono Light</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>ID частоты оплаты</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_stoimUsl">
      <property name="font">
       <font>
        <family>Cascadia Mono Light</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Стоимость услуги</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="lineEdit_stoimUsl">
      <property name="font">
       <font>
        <family>Cascadia Mono Light</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(170, 255, 127);</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEdit_id_opl">
      <property name="font">
       <font>
        <family>Cascadia Mono Light</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(170, 255, 127);</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEdit_id_usl">
      <property name="font">
       <font>
        <family>Cascadia Mono Light</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(170, 255, 127);</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton_open">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>250</y>
     <width>151</width>
     <height>36</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cascadia Mono Light</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(170, 255, 127);</string>
   </property>
   <property name="text">
    <string>Открыть</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_insert">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>151</width>
     <height>36</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cascadia Mono Light</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(170, 255, 127);</string>
   </property>
   <property name="text">
    <string>Добавить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_delete">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>290</y>
     <width>151</width>
     <height>36</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cascadia Mono Light</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(170, 255, 127);</string>
   </property>
   <property name="text">
    <string>Удалить</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_id">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>290</y>
     <width>151</width>
     <height>36</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cascadia Mono Light</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(170, 255, 127);</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
